sympy/geometry/ellipse.py

```diff
@@ -1505,9 +1552,9 @@ def scale(self, x=1, y=1, pt=None):
         c = c.scale(x, y)
         x, y = [abs(i) for i in (x, y)]
         if x == y:
-            return self.func(c, x*self.radius)
+            return self.func(c, x * self.radius)
```

Comment:
Could you revert these changes, please? We follow (mostly) space around `+,-,%` but not `*,**`

Reply:
@smichr  Will do it.